js/npm/sequelize/DataTypes.hx

```diff
@@ -28,14 +28,14 @@ class DataTypes {
 	}
 	/**
 	 * A 32 bit integer. 
-	 * Available properties: `length`, `unsigned:Bool`, `zerofill:Bool`
+	 * Available properties: `length:Int`, `unsigned:Bool`, `zerofill:Bool`
 	 */
 	static public inline function INTEGER(?options:{}):DataTypes {

```

Comment:
Hey, after changing the data types to your implementation i get run-time errors.

Basically it results in different code output, with the inline version it works, the other doesent, it has to do with `(require Sequelize).Datatype(..)` instead of just `Sequelize.Datatype(...)` in the output.

I'm sending a pull request with sequelize tests shortly, you can try and run the tests to see if it can be worked around.